### ISSUE TITLE
The Filter filenames input box is very hard to discover on mobile

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#46999]
* Original report in https://github.com/nextcloud/server/issues/46771 by @szaimen 

The new file list filter input for the file name is hard to reach on mobile (need to open app navigation).

Maybe find a better solution for mobile users.

![image](https://github.com/user-attachments/assets/910c90e9-998d-4603-afc6-910f8e616320)

**Comment by szaimen** [comment]
Just to add some context on this, I think a good solution also for mobile was this: https://github.com/nextcloud/server/pull/41609

But I suppose this is not where we want to go anymore.

However then I guess adding the input box to the main content next to the filter options makes most sense.

**Comment by susnux** [comment]
> However then I guess adding the input box to the main content next to the filter options makes most sense.

This is even worse I would say. You probably do not need it that often but on mobile device proper layout would need to break the filters into multiple lines, and then it takes like 3 rows + chips. That would be a lot of space just for filters on mobile.

**Comment by susnux** [comment]
(from my point of view the only alternative on mobile is https://github.com/nextcloud/server/pull/41609 )

**Comment by jancborchardt** [comment]
Yeah, agree with @susnux here. This is a classic case of where we have so many moving parts in so many apps, that we simply need to go with 1 solution first and have it implemented across the board → and then we can re-evaluate. But if we keep moving stuff around while it is not implemented elsewhere, things will just stay eternally inconsistent.

Hence I would close this issue for now? Or at least move it out of 30.

**Comment by susnux** [comment]
@jancborchardt maybe as a compromise:
1. Keep everything as is
2. On small mobile allow to use the overlaying search box from #41609

This way the desktop user has a nice layout but also mobile users can edit the search while reading the page (no need to open and close the app navigation for editing the current query).

**Comment by jancborchardt** [comment]
@susnux wouldn’t it be a bit strange and/or duplication, as then there is a search both in the nav and on the top? Sure we could also remove it from the nav on mobile, but then it’s inconsistent again.

I would say we need to first make it consistent for a while, also e.g. getting the search into account settings and apps mgmt, then we can see further?
